src/jvm/main/org/jetbrains/kotlinx/lincheck/IdeaPlugin.kt

```diff
@@ -139,6 +140,19 @@ private fun visualize(strategyObject: Any) = runCatching {
     visualizeInstance(testObject, objectToNumberMap, continuationToLincheckThreadIdMap, threadToLincheckThreadIdMap)
 }
 
+/**
+ * This method is called from the trace-debugger evaluation.
+ */
+private fun visualizeTrace(): Array<Any>? = runCatching {
```

Comment:
@eupp should we have a similar fix for the general-purpose model checker? What is the current status of its integration with the plugin?

Reply:
Yes, we would need a similar fix. 
Let's maybe keep `visualizeTrace` name for a while, because I guess it is also used on the trace-debugger plugin side. 

Then once we refactor `visualize` for general-purpose mc, we would be able to re-use it here:
```
private fun visualizeTrace(): Array<Any>? = visualize()
```
or remove it completely.

Reply:
Should we provide thread names through `visualizeTrace`?

Reply:
Also tagging @avpotapov00